File: 2年/CL-21/task-9.cpp
#include <iostream>
#include <ctime>

bool checkPrime(int num) {
    for (int i = 3; i < num; i += 2) {
        if (num % i == 0) {
            return false;
        }
    }
    return true;
}

int main() {
    double st = static_cast<double>(clock()) / CLOCKS_PER_SEC;

    int MaxNum = 100000;
    std::cout << "2 ";
    for (int j = 3; j <= MaxNum; j += 2) {
        if (checkPrime(j)) {
            std::cout << j << " ";
        }
    }
    std::cout << std::endl;

    double en = static_cast<double>(clock()) / CLOCKS_PER_SEC;
    std::cout << "time : " << en - st << " [s]" << std::endl;

    return 0;
}

File: 2年/CL-21/task-9/task_9.py
#!/usr/bin/python3
#
import time


def checkPrime(num):
    for i in range(3, num, 2):
        if num % i == 0:
            return False

    return True


if __name__ == "__main__":
    st = time.time()
    MaxNum = 10000
    print("2 ", end="")
    for j in range(3, MaxNum + 1, 2):
        if checkPrime(j):
            print("{} ".format(j), end="")

    print()
    en = time.time()
    print(en - st)
